crates/context/config/src/client/near.rs

```diff
@@ -0,0 +1,287 @@
+use std::borrow::Cow;
+use std::collections::BTreeMap;
+use std::time;
+
+pub use near_crypto::SecretKey;
+use near_crypto::{InMemorySigner, Signer};
+use near_jsonrpc_client::{methods, JsonRpcClient};
+use near_jsonrpc_primitives::types::query::{QueryResponseKind, RpcQueryResponse};
+use near_jsonrpc_primitives::types::transactions::{RpcTransactionError, TransactionInfo};
+use near_primitives::action::{Action, FunctionCallAction};
+use near_primitives::transaction::{Transaction, TransactionV0};
+pub use near_primitives::types::AccountId;
+use near_primitives::types::{BlockReference, FunctionArgs};
+use near_primitives::views::{
+    AccessKeyPermissionView, AccessKeyView, CallResult, FinalExecutionStatus, QueryRequest,
+    TxExecutionStatus,
+};
+use thiserror::Error;
+use url::Url;
+
+use super::{Operation, Transport, TransportRequest};
+
+#[derive(Debug)]
+pub struct NetworkConfig {
+    pub rpc_url: Url,
+    pub account_id: AccountId,
+    pub access_key: SecretKey,
+}
```

Comment:
New stuff, we should be documenting... in this location most things are fairly self-explanatory, but still good to get into the habit? 🙂

Reply:
There's just no time for this, not as part of the development effort, which is why everything is self-explanatory. We'll revisit at a later time when everything comes together and works.